runtime/rasdump/javadump.cpp

```diff
@@ -1052,14 +1052,30 @@ JavaCoreDumpWriter::writeEnvironmentSection(void)
 	/* Write the running mode data */
 	_OutputStream.writeCharacters("1CIRUNNINGAS   Running as ");
 
+	/* If JITServer enabled, print running mode as JITServer and client UID*/
+#ifdef JITSERVER_SUPPORT
+	if (_VirtualMachine->internalVMFunctions->isJITServerEnabled(_VirtualMachine)) // server
+		_OutputStream.writeCharacters("a JITServer Server");
+	else if (0 == jitConfig->clientUID) // regular jvm
+		_OutputStream.writeCharacters("a standalone");
+	else // client
+		_OutputStream.writeCharacters("a JITServer Client");
 	/* NB : I can't find any code for making this decision in the existing implementation */
-#ifdef J9VM_BUILD_J2SE
+#elif J9VM_BUILD_J2SE
 	_OutputStream.writeCharacters("a standalone");
 #else
 	_OutputStream.writeCharacters("an embedded");
 #endif
 	_OutputStream.writeCharacters(" JVM\n");
 
+#ifdef JITSERVER_SUPPORT
+	if (_VirtualMachine->internalVMFunctions->isJITServerEnabled(_VirtualMachine) && 0 != jitConfig->clientUID) {
```

Comment:
Does this print anything in your tests? I thought that `_VirtualMachine->internalVMFunctions->isJITServerEnabled(_VirtualMachine)` returns `true` only for the server?